content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Installing Ambient Mesh with Helm
+description: How to install Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide shows you how to install Ambient Mesh with Helm.
+Besides the demo in [Getting Started with Ambient Mesh](/docs/ops/ambient/getting-started/),
+we **encourage** you to follow this guide to install Ambient Mesh.
+Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
+
+## Setup Repo Info
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
+
+*See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.*
+
+## Installing the Components
+
+To make it easier to install the components, we have created some Helm value files for you. Make sure you have
+pulled the Helm chart package, or download the [latest version of Istio](/docs/setup/getting-started/#download).
+There will be `ambient-values.yaml` files in the chart package.
+
+In the following steps, we will use the downloaded Istio release package as an example, and make sure you are in the
+root directory of the Istio release package.
+
+### Installing Base Component
+
+The **Base** chart contains the basic CRDs and cluster roles required to set up Istio.
+This should be installed prior to any other Istio component.
+
+{{< text bash >}}
```

Comment:
Unrelated aside: I still think eventually we should/ought to get this down to 2 charts: `istio-dataplane` and `istio-controlplane` - installed separately, upgraded separately.

Reply:
So, in this case, where should `istio-cni` be placed? I thought it was required in ambient to configure data plane, but it seems more like a network helper in the default sidecar mode, which neither belongs to the data plane nor the control plane?

Reply:
Right now `istio-cni` and `ztunnel` are effectively both part of the dataplane.

`istio-cni` isn't part of the dataplane at runtime, but it must be installed with (and upgraded with) the dataplane, in most cases, at the current time.